src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -1,5 +1,7 @@
 package seedu.address.model;
 
+import java.util.ArrayList;
+import java.util.EmptyStackException;
```

Comment:
The code does not need `EmptyStackException` anymore. checkstyle should have caught this.